util/xsl/801.4.dyd.xsl

```diff
@@ -0,0 +1,46 @@
+<!--  This xsl:
+    * rename Dynawo.Electrical.Injectors.InjectorBG_INIT in Dynawo.Electrical.StaticVarCompensators.SVarCStandard_INIT
```

Comment:
Couldn't we merge this one with the ones dealing with dynamic models?

Reply:
see other comment